pontoon/projects/static/js/multiple_project_selector.js

```diff
@@ -0,0 +1,56 @@
+// Contains behaviours of widgets that are shared between admin and end-user interface.
+$(function () {
+  /**
+   * Function keeps track of inputs that contain information about the order of selected projects.
+   */
+  function updateSelectedProjects() {
+    const $selectedList = $('.multiple-project-selector .project.selected'),
+      $selectedProjectsField = $selectedList.find('input[type=hidden]'),
+      selectedProjects = $selectedList
+        .find('li[data-id]')
+        .map(function () {
+          return $(this).data('id');
+        })
+        .get();
+
+    $selectedProjectsField.val(selectedProjects.join());
+  }
+
+  // Choose projects
+  $('body').on(
+    'click',
+    '.multiple-project-selector .project.select li',
```

Comment:
This too doesn't actually need jQuery, but if it's jQuery, it should not be any more complex than necessary.

Same applies to handler below as well.

```suggestion
  $('.multiple-project-selector .project.select li').on(
    'click',
```

Reply:
I believe this will break where the selected element is not in the DOM yet when the handler is attached, e.g. in dashboard tabs.